src/backend/joanie/core/migrations/0025_order_has_consent_to_terms.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.10 on 2024-02-09 15:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0024_siteconfig_siteconfigtranslation'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='order',
+            name='has_consent_to_terms',
+            field=models.BooleanField(default=False, editable=False, help_text='User has consented to the platform terms and conditions.', verbose_name='has consent to terms'),
```

Comment:
This is not aleready in production but you should probably do a data migration setting this property to `True` for order that should have accepted the terms. WDYT ?

Reply:
I wondered if I should do that. But in fact, for existing orders, their owners did not accept terms.

Reply:
But they should.

Reply:
Here is my full thought : 
> They should but they couldn't.

But that's fine for me, I wrote the missing migration.